Bricks/net/http/impl/server.h

```diff
@@ -478,10 +478,9 @@ class GenericHTTPServerConnection final : public HTTPResponder {
  public:
   // The only constructor parses HTTP headers coming from the socket
   // in the constructor of `message_(connection_)`.
-  GenericHTTPServerConnection(
-      Connection&& c,
-      const typename HTTP_REQUEST_DATA::ConstructionParams& params = typename HTTP_REQUEST_DATA::ConstructionParams())
-      : connection_(std::move(c)), message_(connection_, params) {}
+  template <typename... ARGS>
```

Comment:
Well, alternatively we can add two arguments (intial_buffer_size and buffer_growth_k) with same default values as they are in the GenericHTTPRequestData constructor.